css/style.css

```diff
@@ -795,18 +814,25 @@ h2 a {
 #q {
   display: none;
   height: 2.5em;
-  min-width: 100%;
+  max-width: 100%;

```

Comment:
Adjustments to search bar to make it fit. Navbar might be getting a little unwieldy.